modules/3d-tiles/src/tileset/helpers/bounding-volume.js

```diff
@@ -72,21 +65,30 @@ export function createBoundingVolume(boundingVolumeHeader, transform, result) {
 }
 
 function createBox(box, transform, result) {
-  return null;
-
-  const center = new Vector3(box[0], box[1], box[2], scratchCenter);
-  const halfAxes = new Matrix3(); // Matrix3.fromArray(box, 3, scratchHalfAxes);
-
-  // Find the transformed center and halfAxes
-  center = Matrix4.multiplyByPoint(transform, center, center);
-  const rotationScale = Matrix4.getRotation(transform, scratchMatrix);
-  halfAxes = Matrix3.multiply(rotationScale, halfAxes, halfAxes);
+  const center = new Vector3(box[0], box[1], box[2]);
+  let halfAxes = new Matrix3(box.slice(3, box.length));
```

Comment:
You never use this value

Reply:
Not sure what you mean

Reply:
You are assigning `halfAxes` on line 69, and overwriting it on line 73, as far as I can tell, you don't use the value in between.

Reply:
there's a multiplyRight on 82 that uses it

Reply:
There's a multiplyRight on 82 that uses it.

Reply:
Nevermind... I looks to me like it uses the value you assign on line 72. The value you initialize with on line 68 is overwritten and never used.